formats_test.go

```diff
@@ -0,0 +1,181 @@
+package test
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/url"
+	"path"
+	"strconv"
+	"strings"
+	"testing"
+
+	"fmt"
+	"net/http"
+	"os"
+	"sync"
+)
+
+func doRequest(t *testing.T, fnAppName, fnAppRoute string, contentType string, requestBody interface{}) (*bytes.Buffer, *http.Response, error) {
+	u := url.URL{
+		Scheme: "http",
+		Host:   Host(),
+	}
+	u.Path = path.Join(u.Path, "r", fnAppName, fnAppRoute)
+
+	b, err := json.Marshal(requestBody)
+	if err != nil {
+		return nil, nil, err
+	}
+	content := bytes.NewBuffer(b)
+	output := &bytes.Buffer{}
+
+	response, err := CallFN(u.String(), contentType, content, output, "POST", []string{})
+
+	if err != nil {
+		return nil, response, err
+	}
+
+	return output, response, nil
+}
+
+func callMultiple(wg *sync.WaitGroup, times int, t *testing.T, s *SuiteSetup, fnRoute, fnImage,
+	fnFormat string) {
+
+	timeout := int32(30)
+	idleTimeout := int32(10)
+	CreateApp(t, s.Context, s.Client, s.AppName, map[string]string{})
+	CreateRoute(t, s.Context, s.Client, s.AppName, fnRoute, fnImage, "sync",
+		fnFormat, timeout, idleTimeout, s.RouteConfig, s.RouteHeaders)
+
+	wg.Add(times)
+
+	for i := 0; i < times; i++ {
+		go func() {
```

Comment:
CallMultiple should do a sequential test, not a a parallel test (this tests Fn, not the FDK)  - ideally it would be good to have some witness that multiple events were streamed to to the same container?  but I don' think there is a way to safely guarantee this. 

The purpose of this test is to verify that the FDK can handle a sequence of events to the same container (i think)

Reply:
addressed